vendor/brave-ios/Shared/RewardsLogStream.h

```diff
@@ -6,19 +6,29 @@
 
 #import <Foundation/Foundation.h>
 #import <iostream>
+#import <ostream>
 
 #import "bat/ledger/ledger_client.h"
 #import "bat/ads/ads_client.h"
 #import "Logger.h"
 
+class LogStream {
+ public:
+  virtual ~LogStream() = default;
+  virtual std::ostream& stream() = 0;
+};
+
 /// A generic unbuffered logger logs messages via iostream
-class RewardsLogStream : public ledger::LogStream, public ads::LogStream {
+class RewardsLogStream : public ledger::LogStream, public LogStream {
```

Comment:
`public ledger::LogStream` will be removed in https://github.com/brave/brave-browser/issues/6845

Reply:
Is there a reason we need this then? Isn't it the same as `ledger::LogStream`?

Reply:
Removed `LogStream` and will remove `ledger::LogStream` once changes are made to rewards